### PR TITLE
OCPBUGS-33379_2#amending vMotion text

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Version:
4.13+

Issue:
https://issues.redhat.com/browse/OCPBUGS-33379

Link to docs preview:
https://78481--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc#installation-vsphere-installer-infra-requirements_installing-vmc

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@gpei, PTAL

**Comment by gpei** [comment]
@sgaoshang could you please take a look at this?

**Comment by sgaoshang** [comment]
@brendan-daly-red-hat shall we also update `Installing on vSphere` section [here](https://docs.openshift.com/container-platform/4.13/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned) , pls note that from 4.14, we do not have `Installing on VMC` part anymore.

**Comment by sgaoshang** [comment]
``` 
We are aware that VMware's official documentation does not have the name "compute-only vMotion".

  1. Is this a counter explanation to VMware's Storage vMotion?
     If Yes, we assume it means VMware compute vMotion, not VMware Storage vMotion.
     But, we are aware that VMware's official documentation does not have the name "compute-only vMotion".
```
`compute-only vMotion` in doc means VMware compute vMotion, pls see pic

image

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -405,14 +405,16 @@ For more information about creating an account with only the required privileges
...
 
 For more information about vMotion and anti-affinity rules, see the VMware vSphere documentation for  link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vcenterhost.doc/GUID-3B41119A-1276-404B-8BFB-A32409052449.html[vMotion networking requirements] and link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.resmgmt.doc/GUID-FBE46165-065C-48C2-B775-7ADA87FF9A20.html[VM anti-affinity rules].
 --
-* Using Storage vMotion can cause issues and is not supported. If you are using vSphere volumes in your pods, migrating a VM across datastores, either manually or through Storage vMotion, causes invalid references within {product-title} persistent volume (PV) objects that can result in data loss.
+* If you are using vSphere volumes in your pods, migrating a VM across datastores, either manually or through Storage vMotion, causes invalid references within {product-title} persistent volume (PV) objects that can result in data loss.
```
Comment (dfitzmau) [inline modules/installation-vsphere-installer-infra-requirements.adoc:417]:
```suggestion
* If you are using {vmw-full} volumes in your pods, migrating a VM across datastores, either manually or through Storage vMotion, causes invalid references within {product-title} persistent volume (PV) objects that can result in data loss.
```

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

**Comment by brendan-daly-red-hat** [comment]
@sgaoshang, Thanks for your review comments. Could you review the update?

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 2]
```diff
@@ -405,14 +405,16 @@ For more information about creating an account with only the required privileges
 
 If you intend on using vMotion in your vSphere environment, consider the following before installing an {product-title} cluster.
 
-* {product-title} generally supports compute-only vMotion, where _generally_ implies that you meet all VMware best practices for vMotion.
+ * Using Storage vMotion can cause issues and is not supported.
+
+* Using compute-only vMotion to migrate machine workloads is generally supported, where _generally_ implies that you meet all VMware best practices for vMotion.
```
[the remark this answers is not in the document]
Comment (sgaoshang) [reply modules/installation-vsphere-installer-infra-requirements.adoc:410]: I see in vmware doc [here](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-6068ECD7-E3FA-4155-A326-D996BDBDF00C.html), it says "Migrate Your Virtual Machine to a New Compute Resource", not sure if it's official.

**Comment by brendan-daly-red-hat** [comment]
@sgaoshang, I updated the draft text based on comments in https://issues.redhat.com/browse/OCPBUGS-33379. Can you take a look? Thanks.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**